ledger/ledger-offset/src/main/scala/com/daml/ledger/offset/HexOffset.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.offset
+
+import com.daml.lf.data.Ref
+
+object HexOffset {
```

Comment:
I think this behavior probably belongs in `ApiOffset`, as part of the participant, not general offset code.